motioneye/controls/v4l2ctl.py

```diff
@@ -184,17 +175,16 @@ def list_ctrls(device):
 
     output = b''
     started = time.time()
-    cmd = 'v4l2-ctl -d %(device)s --list-ctrls'
-    actual_cmd = cmd % {'device': pipes.quote(device)}
-    logging.debug('running command "%s"' % actual_cmd)
+    cmd = f'v4l2-ctl -d {pipes.quote(device)} --list-ctrls'
+    logging.debug(f'running command "{cmd}"')
 
     try:
-        output = utils.call_subprocess(actual_cmd, shell=True, stderr=subprocess.STDOUT)
+        output = utils.call_subprocess(cmd, shell=True, stderr=subprocess.STDOUT)
```

Comment:
Here it should be possible to set `shell=False` for sake of security. I'll test it.